src/map.cpp

```diff
@@ -532,62 +532,150 @@ bool Map::canThrowObjectTo(const Position& fromPos, const Position& toPos, bool
 	return isSightClear(fromPos, toPos, false);
 }
 
-bool Map::checkSightLine(const Position& fromPos, const Position& toPos) const
+//get a line between two points
+//based on Xiaolin Wu algorithm
```

Comment:
What algorithm is this? Please link to it in the code itself.

Reply:
I was basing that code on this:
https://rosettacode.org/wiki/Xiaolin_Wu%27s_line_algorithm#C.2B.2B